util/fibers/uring_socket.cc

```diff
@@ -378,7 +378,7 @@ uint32_t UringSocket::CancelPoll(uint32_t id) {
 }
 
 void UringSocket::RegisterOnErrorCb(std::function<void(uint32_t)> cb) {
-  DCHECK(error_cb_id_ == UINT32_MAX);
+  DCHECK_EQ(error_cb_id_, UINT32_MAX) << this->fd_;
```

Comment:
```suggestion
  DCHECK_EQ(error_cb_id_, UINT32_MAX) << fd_;
```

Reply:
no need to redirect via `this` :)

Reply:
Oh yes sure, I was just looking what augo-suggestion is offering me on fields